src/applications/personalization/appointments/actions/utils.js

```diff
@@ -29,12 +25,7 @@ const getStagingID = facilityID => {
 const transformAppointment = appointment => {
   const isVideo = appointment.attributes?.kind === 'telehealth';
   const facilityStagingId = getStagingID(appointment.attributes.locationId);
-  const timezone = getTimezoneBySystemId(facilityStagingId)?.timezone;
-  const date = appointment.attributes.start;
-  const startsAt = (timezone
-    ? moment(date).tz(timezone)
-    : moment(date)
-  ).format();
+  const startsAt = appointment.attributes.start;
```

Comment:
Originally the `startsAt` took into account the local system based timezone and adjusted the time based on that, but now we are just using the raw `appointment.attributes.start`

Is this done for a specific reason? I'm not quite following the change here, as it seems like it wouldn't adjust appropriately anymore.

Reply:
I wanted to remove any dependency on the `moment` library, and was trying to look into an equivalent to `moment-tz` but turns out that the [Date object operates on the user's host environment/device](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#date_components_and_time_zones). 

We're using the `startsAt` variable in AppointmentsCard component, where the [raw string is converted to a Date object.](https://github.com/department-of-veterans-affairs/vets-website/blob/My-VA/allison/66858/timezone-cleanup/src/applications/personalization/dashboard/components/health-care/AppointmentsCard.jsx#L10).

Let me know if you have any other ideas!

Reply:
Ok that makes sense and helps me understand what is happening here. Thanks!

:shipit: